### ISSUE TITLE
cannot use it in win10 18362.53

### ISSUE THREAD

**Issue by sbzhaopeng** [stascorp/rdpwrap#764]
the latest version of win10 cannot use rdwrap.

**Comment by yangminxing** [comment]
yes I have the same problem.

**Comment by ner00** [comment]
Try this INI file: https://github.com/stascorp/rdpwrap/issues/738#issuecomment-485195904
According to the feedback the offsets are working.

**Comment by sbzagorodin** [comment]
> Try this INI file: #738 (comment)
> According to the feedback the offsets are working.

That config does not work for me (at least Single User feature). Feedback is wrong.

**Comment by ner00** [comment]
Feedback isn't wrong, there must be another reason for one person claiming success while another claims failure.

Also, why was this closed? 🤨

**Comment by ner00** [comment]
@sbzagorodin paste your termsrv.dll

**Comment by sbzagorodin** [comment]
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/3130313/termsrv.zip)

**Comment by ner00** [comment]
@sbzagorodin Give it a try with [these offsets](https://github.com/stascorp/rdpwrap/issues/766#issuecomment-487979257). I'm not sure it will work, but the addresses seem to match the functions from previous termsrv versions. To be honest, the only offset I got different from the ones in https://github.com/stascorp/rdpwrap/issues/738#issuecomment-485116807 was `LocalOnlyOffset.x64`, I found `82FB5` instead of `759E8`, the rest was exactly the same.
